diff --git a/frontend/karma.conf.js b/frontend/karma.conf.js
index 54a9a44..d031dc8 100644
--- a/frontend/karma.conf.js
+++ b/frontend/karma.conf.js
@@ -36,9 +36,9 @@ module.exports = function (config) {
     port: 9876,
     colors: true,
     logLevel: config.LOG_INFO,
-    autoWatch: true,
-    browsers: ['Chrome'],
-    singleRun: false,
+    autoWatch: false,
+    browsers: ['ChromeHeadless'],
+    singleRun: true,
     restartOnFileChange: true
   });
 };
