diff --git a/iTrust2/src/main/java/edu/ncsu/csc/iTrust2/ITrust2Application.java b/iTrust2/src/main/java/edu/ncsu/csc/iTrust2/ITrust2Application.java
index 8313995..5df3b9c 100644
--- a/iTrust2/src/main/java/edu/ncsu/csc/iTrust2/ITrust2Application.java
+++ b/iTrust2/src/main/java/edu/ncsu/csc/iTrust2/ITrust2Application.java
@@ -5,6 +5,8 @@ import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.orm.jpa.HibernatePropertiesCustomizer;
 import org.springframework.context.annotation.Bean;
 
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
 import edu.ncsu.csc.iTrust2.config.SchemaValidateIntegrator;
 
 /**
@@ -14,7 +16,7 @@ import edu.ncsu.csc.iTrust2.config.SchemaValidateIntegrator;
  *
  */
 @SpringBootApplication ( scanBasePackages = { "edu.ncsu.csc.iTrust2" } )
-public class ITrust2Application {
+public class ITrust2Application extends SpringBootServletInitializer {
 
     /**
      * Main method
@@ -26,6 +28,11 @@ public class ITrust2Application {
         SpringApplication.run( ITrust2Application.class, args );
     }
 
+    @Override
+    protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
+        return builder.sources(ITrust2Application.class);
+    }
+
     /**
      * Used to register a Schema Validator to make sure the database is
      * up-to-date
