diff --git a/iTrust2/pom.xml b/iTrust2/pom.xml
index f799cb1..eacc1e6 100644
--- a/iTrust2/pom.xml
+++ b/iTrust2/pom.xml
@@ -10,6 +10,7 @@
 		<relativePath /> <!-- lookup parent from repository -->
 	</parent>
 	<groupId>edu.ncsu.csc</groupId>
+	<packaging>war</packaging>
 	<artifactId>iTrust2</artifactId>
 	<version>10</version>
 	<name>iTrust2</name>
@@ -37,6 +38,11 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-thymeleaf</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-tomcat</artifactId>
+			<scope>provided</scope>
+		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
@@ -209,6 +215,16 @@
 				</executions>
 			</plugin>
 			
+			<plugin>
+				<groupId>org.apache.tomcat.maven</groupId>
+				<artifactId>tomcat7-maven-plugin</artifactId>
+				<version>2.2</version>
+				<configuration>
+					<url>http://localhost:8080/iTrust2-10</url>
+					<server>TomcatServer</server>
+					<path>/iTrust2</path>
+				</configuration>
+			</plugin>
 			<plugin>
 				<groupId>org.jacoco</groupId>
 				<artifactId>jacoco-maven-plugin</artifactId>
diff --git a/iTrust2/src/main/java/edu/ncsu/csc/iTrust2/ITrust2Application.java b/iTrust2/src/main/java/edu/ncsu/csc/iTrust2/ITrust2Application.java
index 8313995..5df3b9c 100644
--- a/iTrust2/src/main/java/edu/ncsu/csc/iTrust2/ITrust2Application.java
+++ b/iTrust2/src/main/java/edu/ncsu/csc/iTrust2/ITrust2Application.java
@@ -5,6 +5,8 @@ import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.orm.jpa.HibernatePropertiesCustomizer;
 import org.springframework.context.annotation.Bean;
 
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
 import edu.ncsu.csc.iTrust2.config.SchemaValidateIntegrator;
 
 /**
@@ -14,7 +16,7 @@ import edu.ncsu.csc.iTrust2.config.SchemaValidateIntegrator;
  *
  */
 @SpringBootApplication ( scanBasePackages = { "edu.ncsu.csc.iTrust2" } )
-public class ITrust2Application {
+public class ITrust2Application extends SpringBootServletInitializer {
 
     /**
      * Main method
@@ -26,6 +28,11 @@ public class ITrust2Application {
         SpringApplication.run( ITrust2Application.class, args );
     }
 
+    @Override
+    protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
+        return builder.sources(ITrust2Application.class);
+    }
+
     /**
      * Used to register a Schema Validator to make sure the database is
      * up-to-date
