diff --git a/pom.xml b/pom.xml
index b563b66..7161baf 100644
--- a/pom.xml
+++ b/pom.xml
@@ -11,10 +11,12 @@
     <groupId>com.Anvesh</groupId>
     <artifactId>RecepieApp</artifactId>
     <version>0.0.1-SNAPSHOT</version>
+    <packaging>war</packaging>
     <name>RecepieApp</name>
     <description>Demo project for Spring Boot</description>
     <properties>
         <java.version>11</java.version>
+        <start-class>com.anvesh.recepieapp</start-class>
     </properties>
     <dependencies>
         <dependency>
@@ -40,6 +42,11 @@
             <scope>runtime</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
+		    <groupId>org.springframework.boot</groupId>
+		    <artifactId>spring-boot-starter-tomcat</artifactId>
+		    <scope>provided</scope>
+	    </dependency>
         <dependency>
             <groupId>com.h2database</groupId>
             <artifactId>h2</artifactId>
diff --git a/src/main/java/com/anvesh/recepieapp/RecepieAppApplication.java b/src/main/java/com/anvesh/recepieapp/RecepieAppApplication.java
index c7a3ef3..65bd4c1 100644
--- a/src/main/java/com/anvesh/recepieapp/RecepieAppApplication.java
+++ b/src/main/java/com/anvesh/recepieapp/RecepieAppApplication.java
@@ -2,12 +2,18 @@ package com.anvesh.recepieapp;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
 
 @SpringBootApplication
-public class RecepieAppApplication {
+public class RecepieAppApplication extends SpringBootServletInitializer {
 
     public static void main(String[] args) {
         SpringApplication.run(RecepieAppApplication.class, args);
     }
 
+    @Override
+    protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
+        return builder.sources(RecepieAppApplication.class);
+    }
 }
